common/lib/xmodule/xmodule/css/sequence/display.scss

```diff
@@ -292,10 +263,18 @@ $link-color: rgb(26, 161, 222) !default;
 
 nav.sequence-bottom {
   position: relative;
-  width: 79px;
-  height: 1px;
+  height: 45px;
   margin: lh(2) auto;
-  text-align: center;
+  display: flex;
+  justify-content: center;
```

Comment:
IE 11 partially support these 2 declarations, IE 10 not on the list at all. @andy-armstrong what oldest disgraceful IE do we support?

<img width="162" alt="screen shot 2016-09-29 at 3 20 10 pm" src="https://cloud.githubusercontent.com/assets/16127998/18968765/6497c9fa-8658-11e6-8c16-5176cd3406da.png">